### PR TITLE
feat: display horizontal

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes # https://vsf.atlassian.net/jira/software/c/projects/SFUI2/boards/69?modal=detail&selectedIssue=SFUI2-908

# Scope of work

Display horizontal

# Screenshots of visual changes

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+              backgroundColor,
+            )}
+          >
+            <a
+              className="absolute w-full h-full z-1 focus-visible:outline focus-visible:rounded-lg"
+              aria-label={title}
+              href="/"
+            />
+            <div
+              className={classNames('flex items-center justify-between overflow-hidden grow', {
```
Comment (Szymon-dziewonski) [inline apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx:61]:
```suggestion
              className={classNames('flex justify-between overflow-hidden grow', {
```
when removing `items-center` nothing changes, is it needed?

**Comment by justyna-13** [comment]
> question: shouldn't this block be responsive so below some width change orientation to vertical?

It should change that all displays are in column (at least it's on a figma)
Screenshot 2023-04-04 at 13 33 06


https://user-images.githubusercontent.com/46591755/229779578-c835c6f6-5006-481b-823c-4ae9da31f635.mov

File: apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx (hunk 1 above)
Comment (justyna-13) [reply apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx:61]: I'll check items-center and I think overflow-hidden is needed when display has less text lines

Comment (justyna-13) [reply apps/preview/next/pages/showcases/Banners/DisplayHorizontal.tsx]: https://github.com/vuestorefront/storefront-ui/pull/2564/commits/5dcd99203fcebe2bf08f3261d030601c6f0fb320 removed items-center

**Pull request merged**